docker-compose.yml

```diff
@@ -2,22 +2,35 @@ version: '3.3'
 services:
   web:
     build: .
-    command: bash -c "python manage.py collectstatic --noinput && python manage.py runserver 0.0.0.0:8000"
+    command: bash -c "python manage.py collectstatic --noinput && gunicorn ga_core.wsgi:application --bind :8000 --workers 3"
     env_file:
       - /home/jason/github/Genetics_Ark/.env
     volumes:
-      - /var/run/docker.sock:/var/run/docker.sock
-      - /usr/bin/docker:/usr/bin/docker
-      - /home/jason/github/Genetics_Ark/static/tmp:/home/ga/static/tmp
-      - /home/jason/github/Genetics_Ark/DNAnexus_to_igv/jsons:/home/ga/DNAnexus_to_igv/jsons
-      - /home/jason/github/Genetics_Ark/logs:/home/ga/logs
-    ports:
-      - 8000:8000
+      - /var/run/docker.sock:/var/run/docker.sock # to call host docker
+      - /usr/bin/docker:/usr/bin/docker # to call host docker
+      - /home/jason/github/Genetics_Ark/tmp:/home/ga/tmp # zip generated primer designer PDFs
```

Comment:
cause I currently testing it on my computer but it will require changing to server directory e.g. /home/lingj-loc/tmp or something if we test it on dev server